src/content/docs/fr/guides/authentication.mdx

```diff
@@ -188,6 +189,157 @@ const session = await getSession(Astro.request);
 - [`auth-astro` sur GitHub](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#auth-astro)
 - [Documentation Auth.js](https://authjs.dev/)
 
+## Better Auth
+
+Better Auth est un framework indépendant d'authentification (et d'autorisation) pour TypeScript. Il fournit un ensemble complet de fonctionnalités prêtes à l'emploi et comprend un écosystème de plugins qui simplifie l'ajout de fonctionnalités avancées.
```

Comment:
Not sure about my translation here... Would `agnostique` have been better than `indépendant`? It seemed strange to me in this context. So, if you have a better suggestion for `framework-agnostic authentication (and authorization) framework for TypeScript`, I'm all ears! 😄

Reply:
From the [Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/agnostic), "Agnostic" (in computer theme) means "A software that can be used with many different types of platform".
So `indépendant` works perfectly for me. And even if on your usage it has a double sense, it doesn't really matter, we touch the essential.

Reply:
Yeah I know the meaning in English, but I wasn't sure if we also use this term in this sense in French so I was worried that `indépendant` didn't quite capture the definition.
On the other hand, I couldn't find a better word without becoming too wordy compared to the original version. So I preferred to indicate my doubt, it's always better to have a second opinion!
If you think we keep the same meaning, then that's fine with me!